include/mata/parser/inter-aut.hh

```diff
@@ -123,15 +123,16 @@ public:
  * E.g., a formula q1 & s1 will be transformed to a tree with & as a root node
  * and q1 and s2 being children nodes of the root.
  */
-struct FormulaGraph {
+class FormulaGraph {
+public:
     FormulaNode node{};
     std::vector<FormulaGraph> children{};
 
     FormulaGraph() = default;
-    FormulaGraph(const FormulaNode& n) : node(n), children() {}
-    FormulaGraph(FormulaNode&& n) : node(std::move(n)), children() {}
+    explicit FormulaGraph(const FormulaNode& n) : node(n), children() { children.reserve(2); }
```

Comment:
Probably extract the `2` to some named constant.

Reply:
Good idea.